tox.ini

```diff
@@ -1,14 +1,5 @@
 [tox]
-envlist    = py26, py27, py33
+envlist = py26, py27, py32, py33, py34, pypy
```

Comment:
I actually intentionally didn't include py 3.2 since very few actually use it.  (3.4 wasn't out when I last touched this file.)  Do you feel strongly about testing 3.2?